package.json

```diff
@@ -12,8 +12,7 @@
     "format": "prettier --write 'lib/**/*.js' 'bin/**/*.js'",
     "format:ci": "prettier --check 'lib/**/*.js' 'bin/**/*.js'",
     "test": "jest",
-    "version": "auto-changelog -p --template keepachangelog && git add .",
-    "prepublishOnly": "git push --follow-tags && gh-release"
+    "prepublishOnly": "npm ci && run-s lint format:ci test"
```

Comment:
We can follow with https://github.com/netlify/netlify-lambda/issues/361 to improve this